content/white-paper/getting-started/index.md

```diff
@@ -21,11 +21,11 @@ It is also important to take into account the maturity of the project and its si
 
 Another good way to identify a suitable project is to **select those which are actively looking for contributors.** Several websites and tools can help:
 
-* [Awesome for beginners](https://github.com/MunGell/awesome-for-beginners) is a list of projects looking for contributors (beginners or not) by listing their *"Good First Issues"*. In the same way, the website [Up for grabs](https://up-for-grabs.net) makes it possible to find projects in need of contributors.
+* [Awesome for beginners](https://github.com/MunGell/awesome-for-beginners) is a list of projects looking for contributors (beginners or not) by listing their [Good First Issues](https://goodfirstissues.com/). In the same way, the website [Up for grabs](https://up-for-grabs.net) makes it possible to find projects in need of contributors.
```

Comment:
Thanks for adding this. I think we can create a paragraph dedicated to the good first issue website.